mne/io/curry/curry.py

```diff
@@ -0,0 +1,281 @@
+
+                if re.match(regex + " START_LIST", line):
+                    save_lines[regex] = True
+
+    return data_dict
+
+
+def _read_curry_info(fname_base, curry_vers):
+    """Extract info from curry parameter files."""
+    var_names = ['NumSamples', 'NumTrials', 'SampleFreqHz', 'DataFormat',
+                 'SampleTimeUsec', 'NUM_SAMPLES', 'NUM_TRIALS',
+                 'SAMPLE_FREQ_HZ', 'DATA_FORMAT', 'SAMPLE_TIME_USEC']
+
+    param_dict = dict()
+    unit_dict = dict()
+    with open(fname_base + INFO_FILE_EXTENSION[curry_vers]) as fid:
+        for line in iter(fid):
+            if any(var_name in line for var_name in var_names):
+                key, val = line.replace(" ", "").replace("\n", "").split("=")
+                param_dict[key.lower().replace("_", "")] = val
+            for type in CHANTYPES:
+                if "DEVICE_PARAMETERS" + CHANTYPES[type] + " START" in line:
+                    data_unit = next(fid)
+                    unit_dict[type] = data_unit.replace(" ", "") \
+                        .replace("\n", "").split("=")[-1]
+
+    n_samples = int(param_dict["numsamples"])
+    n_trials = int(param_dict["numtrials"])
+    sfreq = float(param_dict["samplefreqhz"])
+    time_step = float(param_dict["sampletimeusec"]) * 1e-6
+    ascii = param_dict["dataformat"] == "ASCII"
+
+    if (sfreq == 0) and (time_step != 0):
+        sfreq = 1. / time_step
+
+    # read labels from label files
+    labels = _read_curry_lines(fname_base + LABEL_FILE_EXTENSION[curry_vers],
+                               ["LABELS" + CHANTYPES[key] for key in
+                                ["meg", "eeg", "misc"]])
+
+    sensors = _read_curry_lines(fname_base + LABEL_FILE_EXTENSION[curry_vers],
+                                ["SENSORS" + CHANTYPES[key] for key in
+                                 ["meg", "eeg", "misc"]])
+
+    all_chans = list()
+    for key in ["meg", "eeg", "misc"]:
+        for ind, chan in enumerate(labels["LABELS" + CHANTYPES[key]]):
+            ch = {"ch_name": chan,
+                  "unit": unit_dict[key],
+                  "kind": FIFFV_CHANTYPES[key]}
+            if key in ("meg", "eeg"):
+                loc = sensors["SENSORS" + CHANTYPES[key]][ind]
+                ch["loc"] = np.array(loc, dtype=float)
+
+            all_chans.append(ch)
+
+    ch_names = [chan["ch_name"] for chan in all_chans]
+    info = create_info(ch_names, sfreq)
+
+    for ind, ch_dict in enumerate(info["chs"]):
+        ch_dict["kind"] = all_chans[ind]["kind"]
+        ch_dict['unit'] = SI_UNITS[all_chans[ind]['unit'][1]]
+        ch_dict['cal'] = SI_UNIT_SCALE[all_chans[ind]['unit'][0]]
+        if ch_dict["kind"] in (FIFF.FIFFV_MEG_CH,
+                               FIFF.FIFFV_EEG_CH):
+            ch_dict["loc"] = all_chans[ind]["loc"]
+
+    return info, n_trials, n_samples, curry_vers, ascii
+
+
+def read_events_curry(fname, event_ids=None):
+    """
+    Read events from Curry event files.
```

Comment:
We usually put the header next to the """
on the first line.

see https://www.python.org/dev/peps/pep-0257/
and mutliline docstrings formatting